layouts/partials/sections/schedule-short.html

```diff
@@ -0,0 +1,20 @@
+<!-- short schedule section started -->
+<section class="schedule-section pb-5 pb-md-0" id="schedule">
```

Comment:
Can you add `style="background-image: none;"`? The snake head looks very weird in that small area.